components/sections/Gallery.jsx

```diff
@@ -15,6 +16,7 @@ export default function Gallery() {
           );
         })}
       </div>
+      <img src="/assets/imgs/default.png" alt="" className="def hidden" />
```

Comment:
Also in the this file where we are mapping the `gallery` array can we change the name of the element in the call-back to something more descriptive than `im` I think `image` or something similar would be better